crates/control/src/ball_filter.rs

```diff
@@ -186,7 +219,7 @@ impl BallFilter {
         );
 
         let filter_parameters = context.ball_filter_configuration;
-        let removed_hypotheses = self.advance_all_hypotheses(
+        let _removed_hypotheses = self.advance_all_hypotheses(
```

Comment:
Should we perhaps just remove this returned `Vec<BallHypothesis>` from all functions returning the removed hypothsis?